03.Rooms-resource.md

```diff
@@ -212,6 +212,7 @@ To join a room you'll need to provide a URI for it. Said URI can represent a Git
 
  - If the room exists and the user has enough permission to access it, it'll be added to the room.
  - If the room doesn't exist but the supplied URI represents a GitHub Org or GitHub Repo the user is an `admin` of, the room will be created automatically and the user added.
+ - If the supplied URI is that of a gitter user (e.g., `/someuser`), the API will create a one-to-one room.
```

Comment:
Move below "If the room exists and the user has enough permission to access it, it'll be added to the room." point